src/scanner/scanner.py

```diff
@@ -261,6 +269,11 @@ def _parse_results(self) -> pd.DataFrame:
         return pd.DataFrame(data)
 
     async def _scan(self, geocode):
+        if geocode not in list(self.data.municipio_geocodigo):
```

Comment:
This code can be moved to the df import in line 339. just add a `where geocode in ...`

Reply:
I could move this to right before the [export](https://github.com/AlertaDengue/episcanner-downloader/blob/fbe335a4c69de2305d2167732e3023b947d6089f/src/scanner/scanner.py#L135), where the error appears, but it would contain the incorrect geocodes if using the scanner without exporting

Reply:
Instead of pulling a list and checking like this, it would be simpler to just check if the geocode's first two digits  corresponds to the UF's numeric code.